samples/bluetooth/central_hr_coded/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(central_hr)
+
+FILE(GLOB app_sources src/*.c)
```

Comment:
There is only one source file. To be consistent with other samples, please change to
```
# NORDIC SDK APP START
target_sources(app PRIVATE
  src/main.c
)
# NORDIC SDK APP END
```